test/unit/core.js

```diff
@@ -464,8 +464,11 @@ QUnit.test( "isNumeric", function( assert ) {
 	assert.ok( t( -16 ), "Negative integer number" );
 	assert.ok( t( 0 ), "Zero integer number" );
 	assert.ok( t( 32 ), "Positive integer number" );
+	assert.ok( t( "0b111110" ), "Binary integer literal" );
```

Comment:
It might be easier to break out the ES6 tests separately and used `QUnit.skip` to skip them. We just need a nice way to detect when we're in an ES6 env or accept the inevitable and look at the userAgent.